tests/Zend/Date/DateObjectTest.php

```diff
@@ -614,6 +614,20 @@ public function testToStringShouldEqualWithAndWithoutPhpFormat()
             $date->toString(DateTime::ATOM, 'php')
         );
     }
+
+    public function testIncompleteTimeData()
+    {
+        try {
+            $date = new Zend_Date();
+            $date->set(0, Zend_Date::TIMES);
+        } catch (\Exception $e) {
```

Comment:
```suggestion
        } catch (Exception $e) {
```

I prefer import classes, but as this class is not namespaced, no need to fqdn